client/my-sites/exporter/advanced-settings.jsx

```diff
@@ -64,10 +65,13 @@ const AdvancedSettings = React.createClass( {
 	}
 } );
 
-const mapStateToProps = ( state ) => {
+const mapStateToProps = ( state, ownProps ) => {
 	const siteId = state.ui.selectedSiteId;
+	const postType = ownProps.postType;
+	const isDateValid = isExportDateValid( state, siteId, postType );
```

Comment:
`isDateValid` actually checks if the date range is valid, perhaps we should update the name in `state/site-settings/exporter/selectors`. Currently the function sounds like it only checks that a single date is valid.